src/autoplex/data/phonons/flows.py

```diff
@@ -393,7 +393,7 @@ def make(
             supercell_matrix_job = reduce_supercell_size_job(
                 structure=structure,
                 min_length=self.supercell_settings.get("min_length", 12),
-                max_length=self.supercell_settings.get("max_length", 25),
+                max_length=self.supercell_settings.get("max_length", 20),
```

Comment:
How is the default max length of 20 determined? Is this a commonly used value in the filed?  If a lattice constant of a unit cell already exceeds 20, what would happen? Would the scale factor be set to 1?

Reply:
the length of 20 appears to cause the least problems for the run time and yields good supercells. If a lattice parameter of a unit cells exceeds this value, it won't be scaled and stay like it is. But for the phonon part, so large unit cells are not intended to use (on a regular basis).